paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/StoreCommitter.java

```diff
@@ -60,8 +64,9 @@ public ManifestCommittable combine(
     }
 
     @Override
-    public void commit(List<ManifestCommittable> committables)
+    public void commit(List<ManifestCommittable> committables, OperatorIOMetricGroup metricGroup)
             throws IOException, InterruptedException {
+        metricGroup.getNumBytesOutCounter().inc(calcDataBytesSend(committables));
```

Comment:
Count this only after the commit is successful. Otherwise if the commit is retried through fail over, this will be counted multiple times.